src/components/App.js

```diff
@@ -225,7 +198,7 @@ const App = () => {
   }
 
   if (elevationVisible && elevationProfile) {
-    mapHeight -= elevationHeight;
+    mapHeight -= ELEVATION_HEIGHT;
```

Comment:
I wonder if this was previously non-constant in state with some sort of functionality in mind for resizing the elevation widget.

Reply:
That's what I expected but the `setElevationHeight` state method wasn't being used anywhere.